effects.cwt

```diff
@@ -2944,3 +2964,14 @@ alias[effect:set_looter_hostility_days] = {
     who = scope[character]
     days = int
 }
+
+## scope = character
+### True if the scoped character has a strong claim on the given title.
+alias[effect:has_strong_claim] = scope[title]
+
+# TODO: Doc
+## scope = character
+### 
+alias[effect:attacker] = {
```

Comment:
Not sure that's a general effect as opposed to something local to wherever that's used.

Reply:
It's used in decisions and events.

Reply:
Is this not a link?

Reply:
It is yea, I'll move them.

Reply:
Well it's an  effect not a scope change.

Reply:
Is it not a scope change? From what I understand it is just a scope change to another character (that is, the attacker).

Reply:
Well no, here it's used to apply an effect on the attacker

Reply:
Let me look at removing it.